src/WebJobs.Script/Utility.cs

```diff
@@ -101,6 +101,23 @@ internal static Attribute GetHierarchicalAttributeOrNull(MethodInfo method, Type
             return null;
         }
 
+        internal static string GetDebugEngineInfo(RpcWorkerConfig workerConfig, string runtime)
+        {
+            if (runtime == RpcWorkerConstants.DotNetIsolatedLanguageWorkerName)
+            {
+                if (workerConfig.Description.DefaultRuntimeVersion.Contains(RpcWorkerConstants.NetFramework))
+                {
+                    return RpcWorkerConstants.NetCoreDebugEngine;
```

Comment:
Shouldn't this be `NetFrameworkDebugEngine` and the else statement should be NetCore?

Reply:
yes it should be